CSS/syntax_test_css.css

```diff
@@ -507,6 +507,9 @@
     color: #137;
 /*         ^^^^ constant.other.color.rgb-value.css */
 
+    color: #0f0a;
+/*         ^^^^^ constant.other.color.rgb-value.css */
```

Comment:
This should probably be `constant.other.color.rgba-value.css` like the 8-char version below.